service/rpc/servicer.go

```diff
@@ -419,6 +419,320 @@ func (s *Servicer) BlockTransaction(ctx context.Context, request *types.BlockTra
 	}, nil
 }
 
+func (s *Servicer) Call(
+	ctx context.Context,
+	callRequest *types.CallRequest,
+) (*types.CallResponse, *types.Error, error) {
+	switch callRequest.Method {
+	case isAccount:
+		address, ok := callRequest.Parameters["address"].(common.Address)
```

Comment:
I also debated strongly typing the inputs to each of these and using `types.UnmarshalMap` to unmarshal to a specific struct for each.